drivers/snmp-ups.c

```diff
@@ -957,18 +957,6 @@ void nut_snmp_init(const char *type, const char *hostname)
 			g_snmp_sess.securityAuthProto = usmHMACSHA1AuthProtocol;
 			g_snmp_sess.securityAuthProtoLen = sizeof(usmHMACSHA1AuthProtocol)/sizeof(oid);
 		}
-		else if (strcmp(authProtocol, "SHA256") == 0) {
```

Comment:
probably an issue here with the revert of these addition (probably done on FTY only)

Reply:
Exactly on FTY - they got added twice from two developments:
* original on FTY, with no care for portability
* included upstream, improved (ifdefs et al) and merged back as part of master

Somehow this routine ended up having both spellings of the codebase, so here a commit fixes the error.

Otherwise, FTY branch still does not build on agents with older libnetsnmp that does not have these symbols, and generally needlessly differs from upstream NUT.